scipy/stats/_continuous_distns.py

```diff
@@ -6123,6 +6146,21 @@ def _stats(self, b, moments='mv'):
     def _entropy(self, c):
         return 1 + 1.0/c - np.log(c)
 
+    def fit(self, data, *args, **kwds):
+        parameters = _check_fit_input_parameters(self, data, args, kwds)
+        data, fshape, floc, fscale = parameters
+        if floc is None:
+            raise RuntimeError("`floc` must be fixed to obtain a "
+                               "well defined solution.")
+        if np.any(data - floc < (fscale if fscale else 0)):
```

Comment:
```suggestion
        if np.any(data - floc < (fscale if fscale else 1)):
```
Right?
(How else would `fscale` get on the right side of the inequality?)

Reply:
I'm not sure I agree, as `fscale`'s bounds is that it must be greater than 0. 



> (How else would fscale get on the right side of the inequality?)

SciPy says that `x >= 1`, where after adding loc and scale, `y = (x - loc) / scale`. 

``` 
y >= 1
y = (x - loc) / scale
1 <= (x - loc) / scale
scale <= x - loc
scale > 0

so 
 
(x-loc) > 0  
```

Please correct me if I am wrong, but I thought that was the explanation for how SciPy describes the range and how the textbook described it: 
![Screen Shot 2020-07-22 at 3 30 33 PM](https://user-images.githubusercontent.com/44255917/88235712-4963df80-cc30-11ea-8ef4-0d8d0c93c7c9.png)


If the user has chosen a nonzero  `floc` to translate values, all values must be above or equal to `fscale`, which is analytically defined as the `data.min()`. 

So, in the case where `fscale` is not fixed, it cannot be calculated until all of the data has been shifted according to `floc`. But we know that `fscale` must be greater than zero, so that is the minimum bound for the data. 




-- I also notice that I need to use <= instead of < here.

Reply:
Ah, you're right. It makes a lot of sense when you consider that `fscale` is the minimum value of x-loc: that ensures that y = (x-loc)/scale >= 1.